python/lsst/sims/maf/plots/spatialPlotters.py

```diff
@@ -230,7 +230,7 @@ def __call__(self, metricValue, slicer, userPlotDict, fignum=None):
         plotDict.update(userPlotDict)
         if plotDict['scale'] is None:
             plotDict['scale'] = (hp.nside2pixarea(slicer.nside, degrees=True) / 1000.0)
-        fignum = self.baseHist(metricValue, slicer, plotDict, fignum=fignum)
+        fignum = self.baseHist(metricValue[~metricValue.mask], slicer, plotDict, fignum=fignum)
```

Comment:
This is in the histogram section? This is also already fixed, by using only the compressed metricValue in the baseHist itself (so the full metricValue masked array should still be sent, otherwise things will fail in baseHist).